crypto/dilithium/p_pqdsa_test.cc

```diff
@@ -1182,11 +1187,6 @@ TEST_P(PQDSAParameterTest, KeyGen) {
   EXPECT_EQ(ERR_LIB_EVP, ERR_GET_LIB(err));
   EXPECT_EQ(EVP_R_UNSUPPORTED_ALGORITHM, ERR_GET_REASON(err));
   ctx.get()->pkey = (EVP_PKEY*) tmp;
-
-  // attempt to change NID after PKEY is set and verify that this null's the key
-  ASSERT_TRUE(EVP_PKEY_pqdsa_set_params(pkey.get(), nid));
-  ASSERT_TRUE(pkey.get()->pkey.pqdsa_key->public_key == nullptr);
-  ASSERT_TRUE(pkey.get()->pkey.pqdsa_key->private_key == nullptr);
```

Comment:
I removed this check because to run it, it requires exposing `EVP_PKEY_pqdsa_set_params`. Currently, this function isn't exposed, so changing the NID after the PKEY is set is not something that is possible -- by exposing the function to run the test, we would make such behaviour possible.